test/test_other.py

```diff
@@ -15316,8 +15314,8 @@ def test_SUPPORT_BIG_ENDIAN(self):
     'noexcept': ['-fno-exceptions'],
     'default': [],
     'except': ['-sDISABLE_EXCEPTION_CATCHING=0'],
-    'except_wasm': ['-fwasm-exceptions', '-sWASM_LEGACY_EXCEPTIONS'],
-    'except_wasm_exnref': ['-fwasm-exceptions', '-sWASM_LEGACY_EXCEPTIONS=0']
+    'except_wasm': ['-fwasm-exceptions', '-sWASM_LEGACY_EXCEPTIONS=0'],
```

Comment:
Can you just drop the `'-sWASM_LEGACY_EXCEPTIONS=0'` here maybe?

Reply:
This is meant to be Wasm (exnref) so without that it will use the legacy exception instead, no? (`WASM_LEGACY_EXCEPTIONS` is on by default now)

Reply:
Oh I didn't realize that part.  lgtm